omnibus/files/private-chef-cookbooks/private-chef/templates/default/openssl.cnf.erb

```diff
@@ -0,0 +1,361 @@
+#
+# OpenSSL example configuration file.
+# This is mostly being used for generation of certificate requests.
+#
+
+# This definition stops the following lines choking if HOME isn't
+# defined.
+HOME			= .
+RANDFILE		= $ENV::HOME/.rnd
+
+# Extra OBJECT IDENTIFIER info:
+#oid_file		= $ENV::HOME/.oid
+oid_section		= new_oids
+
+<% if node['private_chef']['fips_enabled'] -%>
```

Comment:
I don't think this will work like we want in the case of turning fips mode on and back off.  Since once you turn fips_enabled to false, this template file won't get updated:

https://github.com/chef/chef-server/blob/master/omnibus/files/private-chef-cookbooks/private-chef/recipes/default.rb#L40-L42

I think we may want a fips_disabled recipe that deletes this entire file when fips isn't on.